cms_static.py

```diff
@@ -2,6 +2,7 @@
 GH_CMSSW_REPO    = 'cmssw'
 GH_CMSDIST_REPO  = 'cmsdist'
 BUILD_REL        = '^[Bb]uild[ ]+(CMSSW_[^ ]+)'
+CREATE_REPO      = '^[Cc]reate[ ]+repository[ ]+([A-Za-z0-9]+)[-/]([A-Za-z0-9]+)'
```

Comment:
package regexp should be something like  `^[A-Z][0-9A-Za-z]+[-/][a-zA-Z][0-9A-Za-z]+` (https://github.com/cms-sw/cms-bot/blob/master/process_pr.py#L66)